bundles/AdminBundle/Resources/public/js/pimcore/object/tags/time.js

```diff
@@ -26,19 +26,29 @@ pimcore.object.tags.time = Class.create(pimcore.object.tags.abstract, {
     },
 
     getLayoutEdit: function () {
-        this.component = new Ext.form.TimeField({
+        const options = { 
             fieldLabel: this.fieldConfig.title,
             format: "H:i",
             emptyText: "",
-            width: 200,
+            width: this.fieldConfig.width ? this.fieldConfig.width : 250,
             value: this.data,
             allowBlank: (!this.fieldConfig.mandatory),
             minValue: (this.fieldConfig.minValue) ? this.fieldConfig.minValue : null,
             maxValue: (this.fieldConfig.maxValue) ? this.fieldConfig.maxValue : null,
             componentCls: this.getWrapperClassNames(),
             increment: (this.fieldConfig.increment) ? this.fieldConfig.increment : 15
-        });
+        };
+    
+        if (this.fieldConfig.labelWidth) {
```

Comment:
All those settings (`width`, `labelWidth` and `labelAlign`) are not available in the fields configuration, so that won't work. 
In order to get this to work we'd need to add them here as well: 
https://github.com/pimcore/admin-ui-classic-bundle/blob/1976b8f869ad450e1b3d2b9e4aebcc47f3487e7e/public/js/pimcore/object/classes/data/time.js#L82
https://github.com/pimcore/pimcore/blob/53449686033cdb9dd10f5c64c9636b8c725e2a35/models/DataObject/ClassDefinition/Data/Time.php#L46